src/test/java/com/wikia/webdriver/testcases/adstests/TestTopWamWikis.java

```diff
@@ -11,7 +11,7 @@
 public class TestTopWamWikis extends TemplateNoFirstLoad {
 
   Credentials credentials = Configuration.getCredentials();
-  Integer numberOfTop1kWikis = 997;
+  Integer numberOfTop1kWikis = 999;
```

Comment:
You need this only if you are intrested in "what is number of wikis marked by top1k param. So if future somebody will run some experiment base on this param and be suprised that there is not enough  results - he will need to manually check this.

Plus - how its possible that this number increased? Did you again start marking wikis with this param?

My suggestion - ask Nef about dropping top1k functionality (or moving it to Life Support) and remove tests if he agree.